_data/simple-icons.json

```
@@ -2640,6 +2640,11 @@
             "hex": "000000",
             "source": "https://blog.society6.com/app/themes/society6/dist/images/mark.svg"
         },
+        {
+            "title": "Socket.io",
+            "hex": "010101",
+            "source": "https://socket.io/css/images/logo.svg"
```

Since the SVG is used on the homepage and this specific path might easily change in the future, can you change the source URL?

```suggestion
            "source": "https://socket.io"
```